client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -238,7 +237,8 @@ const Checkout = React.createClass( {
 				}
 			}
 		} = this.props;
-		const redirectPath = this.getCheckoutCompleteRedirectPath();
+		const redirectPath = this.getCheckoutCompleteRedirectPath(),
+			{ receipt_id: receiptId } = receipt;
```

Comment:
Are you sure a receipt will always be returned by the API? We do currently do a presence check for it. My understanding is that a receipt is sometimes failed to be created, which is why we have an optional receipt ID in `/checkout/thank-you/:site/:receiptId?`.